ngrinder-controller/build.gradle

```diff
@@ -2,48 +2,49 @@ description = "ngrinder-controller"
 
 buildscript {
     repositories {
-        maven { url "http://repo.spring.io/libs-snapshot" }
+        maven { url "https://repo.spring.io/libs-snapshot" }
         maven { url "https://repo.spring.io/libs-milestone" }
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.0.6.RELEASE")
     }
-}
-
-configurations {
-    executableWarDeps
 }
 
 apply plugin: "war"
+apply plugin: "org.springframework.boot"
 
 sourceSets.main.output.resourcesDir = sourceSets.main.output.classesDir
 
+def agentScriptPath = "${sourceSets.main.output.resourcesDir}/ngrinder-sh/agent"
+def monitorScriptPath = "${sourceSets.main.output.resourcesDir}/ngrinder-sh/monitor"
+
 dependencies {
-    compile project(":ngrinder-starter")
     compile project(":ngrinder-core")
-    compile project(":ngrinder-sh")
     compile files("${rootDir}/lib/universal-analytics-java-1.0.jar")
+
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-freemarker', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: springboot_version)
+    compile (group: 'org.springframework.boot', name: 'spring-boot-loader', version: springboot_version)
+
     compile (group: "rome", name: "rome", version: "1.0")
     compile (group: "com.ibm.icu", name: "icu4j", version: "4.6")
     compile (group: "sonia.svnkit", name: "svnkit-dav", version: svnkit_version)
     compile (group: "sonia.svnkit", name: "svnkit", version: svnkit_version)
     compile (group: "javax.servlet.jsp", name: "jsp-api", version: "2.1")
```

Comment:
Do we need to import jsp-api explicitly here?

Reply:
This seems to be necessary to use  `JspTaglibs` at freemarker.

Reply:
OKAY.